apps/examples/lcd_test/example_lcd.c

```diff
@@ -52,17 +52,11 @@
 
 #include <tinyara/config.h>
 #include <tinyara/lcd/lcd_dev.h>
-#if defined(CONFIG_TOUCH)
```

Comment:
Let's say that touch screen example is splitted from the lcd_test in the commit description.